docs/source/model-registry.rst

```diff
@@ -78,47 +79,169 @@ version to one of the other valid stages.
 
 
 API Workflow
-------------
+============
+
+An alternative way to interact with Model Registry is using the `MLflow model flavor <https://www.mlflow.org/docs/latest/python_api/index.html>`_ or `MLflow Client Tracking API <https://www.mlflow.org/docs/latest/python_api/mlflow.tracking.html>`_ interface.
+In particular, you can register a model during an MLflow experiment run or after all your experiment runs.
 
-You can register a model using two APIs: ``register_model`` and ``log_model``.
+Adding an MLflow Model to the Model Registry
+--------------------------------------------
 
-If the model has already been logged, you can add it to the registry with the ``register_model`` API, referencing the model by its URI.
+There are three programmatic ways to add a model to the registry. First, you can use the ``mlflow.<model_flavor>.log_model(...)`` method. For example, in your code:
 
 .. code-block:: py
 
-   mlflow.register_model("runs:/e148278107274..832/artifacts/model", "EmailMarketingCampaign")
+    with mlflow.start_run(run_name="YOUR_RUN_NAME") as run:
+        params={"n_estimators": 5, "random_state": 42}
+        sk_learn_rfr=RandomForestRegressor(params)
+        # Log parameters and metrics using the MLflow APIs
+        mlflow.log_params(params)
+        log_param("param_1", randint(0, 100))
+        log_metric("metric_1", random())
+        log_metric("metric_2", random() + 1)
+       …
+       # Log the sklearn model and register as version 1
+       mlflow.sklearn.log_model(sk_model=sk_learn_rfr,
+                artifact_path="sklearn-model",
+                registered_model_name="sk-learn-random-forest-reg-model")
```

Comment:
This `...` are unrecognized characters.

Reply:
Ah, no wonder it won't highlight the strings... was wondering.
Will remove it.